package.json

```diff
@@ -33,7 +33,7 @@
     "@strapi/utils": "^5.8.0",
     "axios": "^1.8.4",
     "generate-password": "^1.7.1",
-    "pkce-challenge": "^3.1.0",
```

Comment:
Merging all these Pull Requests will allow us to remove pkce-challenge.

* #88
* #89 
* #90

When reviewing the changes on this branch locally, please checkout the following commit.

* 4d68110b527d5b3da0de55c66dc12f1e1cb172f4